includes/Pro_Upgrades.php

```
@@ -76,6 +77,48 @@ public function add_to_others_fields( $fields ) {
+                'pricing_checkbox',
+                'pricing_dropdown',
+                'pricing_multiselect',
+                'cart_total',
+            ],
+        ];
+
+        // Insert Pricing Fields after Custom Fields (position 2-3 depending on taxonomies)
+        $new_sections = [];
+        $inserted = false;
+
+        foreach ( $sections as $index => $section ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$index'. (undefined)

(UnusedLocalVariable)